manager/container.go

```diff
@@ -356,7 +372,7 @@ func (self *containerData) nextHousekeeping(lastHousekeeping time.Time) time.Tim
 		}
 	}
 
-	return lastHousekeeping.Add(self.housekeepingInterval)
+	return lastHousekeeping.Add(jitter(self.housekeepingInterval, JitterFactor))
```

Comment:
Makes sense. Wrap `cont.Start()` inside of go thread then?

On Fri, Jan 15, 2016 at 1:30 PM, Timothy St. Clair <notifications@github.com
> wrote:

> In manager/container.go
> <https://github.com/google/cadvisor/pull/1057#discussion_r49907396>:
>
> > @@ -356,7 +372,7 @@ func (self *containerData) nextHousekeeping(lastHousekeeping time.Time) time.Tim
> >  		}
> >  	}
> >
> > -	return lastHousekeeping.Add(self.housekeepingInterval)
> > +	return lastHousekeeping.Add(jitter(self.housekeepingInterval, JitterFactor))
>
> I think your initial starting point should also have jitter in the case of
> initial burst of containers.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/1057/files#r49907396>.
>